ui/ui_streamlit.py

```diff
@@ -0,0 +1,1096 @@
+import datetime
+import os
+import sys
+import streamlit as st
+import re
+from st_aggrid import AgGrid, GridOptionsBuilder
+import pandas as pd
+from streamlit_drawable_canvas import st_canvas
+from PIL import Image, ImageDraw
+from streamlit_player import st_player
+from youtube_dl import YoutubeDL
+import cv2
+from base64 import b64encode
+import json
+from tempfile import NamedTemporaryFile
+import match_folder_structure_validator
+from zipfile import ZipFile
+from pathlib import Path
+from execute_scrapper import run_script
+from read_team_options import read_teams_options
+import streamlit_authenticator as stauth
+import database as db
+from footballdatabase_eu_scrapper import get_data_from_GUI
```

Comment:
please write imports in order, 1. built-ins 2.third party lbraries 3.this project modules
also firstly from ... import ..., then import ...